bitnami/kafka/templates/NOTES.txt

```diff
@@ -23,29 +23,29 @@ Kafka can be accessed via port 9092 on the following DNS name from within your c
 To create a topic run the following command:
 
     export POD_NAME=$(kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ template "kafka.name" . }},app.kubernetes.io/instance={{ .Release.Name }},app.kubernetes.io/component=kafka" -o jsonpath="{.items[0].metadata.name}")
-    kubectl exec -it $POD_NAME -- kafka-topics.sh --create --zookeeper {{  template "kafka.zookeeper.fullname" . }}:2181 --replication-factor 1 --partitions 1 --topic test
+    kubectl -n {{ .Release.Namespace }} exec -it $POD_NAME -- kafka-topics.sh --create --zookeeper {{  template "kafka.zookeeper.fullname" . }}:2181 --replication-factor 1 --partitions 1 --topic test
```

Comment:
Can you use the long flag `--namespace` instead of the short one? Same for the rest of lines.

Reply:
done